package.json

```diff
@@ -11,6 +11,7 @@
     "build:content:test": "node script/test-cms-export.js",
     "build:webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/webpack.config.js --progress",
     "cy:test:circle": "./script/run-cypress-tests-circle.sh",
+    "cy:test:docker": "cypress run --config baseUrl=http://vets-website:3001",
```

Comment:
Don't we also want to pass the globbed pattern to match the tests?

Reply:
```suggestion
    "cy:test:docker": "cypress run --config baseUrl=http://vets-website:3001 --spec 'src/**/tests/**/*.cypress.spec.js'",
```

Reply:
Wouldn't the config in `cypress.json` be used for determining the `testFiles`?

Reply:
Oh, that's right. I forgot that was configured in there.